src/screens/TaskListScreen.tsx

```diff
@@ -0,0 +1,725 @@
+import React, { useState } from 'react';
```

Comment:
Some of this could be broken into components. can you do that with react native if its not a screen?